CredentialManager/README.md

```diff
@@ -48,7 +48,7 @@ This sample app implements the following use cases :
 * Android Studio
 * Java 11
 * Kotlin plugin version : 1.8.10.
-* The sample app uses Credential Manager API version 1.0.0-alpha06
+* The sample app uses Credential Manager API version 1.0.3-alpha01
```

Comment:
I think it should be 1.3.0-alpha01
https://github.com/android/identity-samples/blob/main/CredentialManager/app/build.gradle#L104C54-L104C67

Reply:
ah yes, typo...thanks @ksemenova